build.gradle.kts

```diff
@@ -215,7 +215,7 @@ jacoco {
 if(!Os.isFamily(Os.FAMILY_WINDOWS)) {
     opa {
         srcDir = "$rootDir/src/main/resources/policies"
-        testDir = "$rootDir/src/main/resources/policies/tests"
+        testDir = "$rootDir/src/test/opa/policies"
```

Comment:
I would prefer if we keep the test "path" the same as the main "path", so it is obvious where to find tests related to the main sources.

Reply:
Moved in `resources`
This will have slight implications though as tests will be shown as resources in IntelliJ searches.